topology/examples/ctx_2n.py

```diff
@@ -40,22 +40,26 @@
 
 nest.CopyModel('iaf_neuron', 'pyr')
 nest.CopyModel('iaf_neuron', 'in')
-ctx = topo.CreateLayer({'columns': 4, 'rows': 3,
+col = 4
+row = 3
+el = ['pyr', 'in']
+ctx = topo.CreateLayer({'columns': col, 'rows': row,
                         'extent': [2.0, 1.5],
-                        'elements': ['pyr', 'in']})
+                        'elements': el})
 
-nest.PrintNetwork()
+nest.PrintNetwork(1, (0,))
 
-nest.PrintNetwork(2)
+nest.PrintNetwork(2, (0,))
 
 nest.PrintNetwork(2, ctx)
```

Comment:
I think we should use the opportunity here to improve the example. All these different `PrintNetwork` calls make very limited sense, and `PrintNetwork` will change anyways in NEST3. So I would reduce it to just one call to
```
nest.PrintNetwork(2)
```
I we then suppress the deprecation warning for `CurrentSubnet()` in `PrintNetwork()`, all should work nicely.